Source/WebKit/UIProcess/mac/WKFullScreenWindowController.mm

```diff
@@ -609,7 +609,7 @@ - (void)clearVideoPresentationManagerObserver
 
 - (void)setVideoPresentationManagerObserver
 {
-    auto* videoPresentationManager = self._videoPresentationManager;
+    RefPtr<WebKit::VideoPresentationManagerProxy> videoPresentationManager = self._videoPresentationManager;
```

Comment:
I am assuming `RefPtr` didn't work?

Reply:
Yeah, it doesn't work. Something to do with Objective-C++.